### ISSUE TITLE
HomeKit never finds accessory?

### ISSUE THREAD

**Issue by deadrubberboy** [koush/scrypted#66]
Hello,

Just installed Scrypted via Docker on MacMini 

Up and running easy peasy! Great work.

Added the HomeKit plugin. Scanning the QR code in Home.app on iPhone shows “add this bridge” and then it just hangs there. Tried typing the code and same thing.

Am I doing something wrong?

**Comment by koush** [comment]
Try manually setting the address override in the HomeKit plugin to your ethernet address.

**Comment by koush** [comment]
Oh, this is docker on Mac. It won't work since how networking on docker for Mac does not work. The host network is actually  for the linux VM host, not the Mac host.

**Comment by deadrubberboy** [comment]
Aww dang.

Thanks for the quick reply! I have a RasPi running HomeBridge (and that’s it. Used their official image and instructions). Any way to utilize that? Either native HomeBridge or also run Srypted on it? It’s a pi4.

My hesitation to native is that I used to have all the *arrs and other things native and the dependencies conflicted and the whole thing became a mess I’m not smart enough to entangle. Started fresh when I discovered Docker and I like that they are all siloed.
